packages/netlify-cms-core/src/actions/config.ts

```diff
@@ -386,10 +386,10 @@ async function getConfigYaml(file: string, hasManualConfig: boolean) {
   const contentType = response.headers.get('Content-Type') || 'Not-Found';
   const isYaml = contentType.indexOf('yaml') !== -1;
   if (!isYaml) {
+    console.log(`Response for ${file} was not yaml. (Content-Type: ${contentType})`);
     if (hasManualConfig) {
       return {};
     }
-    throw new Error(`Response for ${file} was not yaml. (Content-Type: ${contentType})`);
```

Comment:
We still want to attempt parse the configuration in this case.
For example, when you deploy to S3 via the `aws-cli` it sometimes fails to set the correct mime type for files. See https://github.com/aws/aws-cli/issues/1249

Reply:
Alright, got it